chia/wallet/did_wallet/did_wallet_puzzles.py

```diff
@@ -114,15 +90,6 @@ def is_did_innerpuz(inner_f: Program) -> bool:
     return inner_f == DID_INNERPUZ_MOD
 
 
-def is_did_core(inner_f: Program) -> bool:
```

Comment:
Glad to see the back of this function.
Once upon a time the singleton layer was called the did core, which is why this function is called that. 
Hopefully that's the last of references to did_core